README.md

```diff
@@ -106,88 +106,109 @@ On Windows, in Window > Preferences > General > Workspace, set "Text file encodi
-* check the "Use the Eclipse code formatter" option and use [sonar-formatter.xml](/eclipse/sonar-formatter.xml) as the Eclipse Java Formatter config file
-* check the "From file" option in the "Import order" section and use [sonar.importorder](/eclipse/sonar.importorder)
+### Eclipse Code Formatter
+Intellij IDEA users must install the plugin [Adapter for Eclipse Code Formatter](http://plugins.jetbrains.com/plugin/?id=6546):
+* Check `Use the Eclipse code formatter`
+* Set `Eclipse workspace/project folder or config file` to [sonar-formatter.xml](/eclipse/sonar-formatter.xml)
+* Check `Optimize Imports (IntelliJ's Import Optimizing must be turned ON)`
+* Set `Import Order from file` to [sonar.importorder](/eclipse/sonar.importorder)
+![Intellij code style](intellij/intellij-eclipse-formatter-config.png)
 
-You can either import the code style settings to IntelliJ IDEA by simply importing [this scheme](intellij/codestyle_sonar_developer_toolset.xml)
-in IDEA's code style settings (`Editor > 'Code Style' > Java > 'Scheme:' > ⚙️  > 'Import Scheme' > 'IntelliJ IDEA code style XML'`), or you can apply the settings manually (see below). The result should be the same.
+### General Editor
+Go to `Preferences/Settings > Editor > General`:
+* Check `Ensure every saved file ends with a line break` (under the `On Save` section).
 
-![Intellij code style](intellij/intellij-code-style.png)
+Go to `Preferences/Settings > Editor > General > Auto import`:
+* Check `Optimize imports on the fly` for Java:
 
-Go to `Preferences/Settings > Editor > General` and check the option `Ensure every saved file ends with a line break` (under the `On Save` section).
+![image](https://user-images.githubusercontent.com/50145663/135304815-0b51f1da-16d4-4fc7-a278-a13067e78d7c.png)
 
-Go to `Preferences/Settings > Editor > Code Style > Java > Wrapping and Braces > Method declaration parameter` and uncheck `Align when multiline`
+### Automatic IntelliJ Code Style
+You can either import the code style settings to IntelliJ IDEA by simply importing [this scheme](intellij/codestyle_sonar_developer_toolset.xml) in IDEA's code style settings (`Editor > 'Code Style' > Java > 'Scheme' > ⚙️  > 'Import Scheme' > 'IntelliJ IDEA code style XML'`),
+or you can apply the settings manually (see below). The result should be the same.
 
-Go to `Preferences/Settings > Editor > Code Style > Java > Tabs and Indents` and update:
-* `Tab size: 2`
-* `Indent: 2`
-* `Continuation indent: 2`
+### Manual Java Code Style
+Follow these steps to set code style settings if not using [this scheme](intellij/codestyle_sonar_developer_toolset.xml) from above.
 
-Go to `Preferences/Settings > Editor > Code Style > Java > JavaDoc > Other` and uncheck `Generate "<p>" on empty lines`
+Go to `Preferences/Settings > Editor > Code Style > Java > Tabs and Indents`:
+* Set `Tab size` to 2
+* Set `Indent` to 2
+* Set `Continuation indent` to 2
```

Comment:
I changed the code style lists to start with a verb. Also, I changed the embedded lists to be proper bullet lists.

More information here: https://developers.google.com/tech-writing/one/lists-and-tables#:~:text=An%20embedded%20list%20(sometimes%20called%20a%20run%2Din%20list)